src/modules/add-display.js

```diff
@@ -1,27 +1,80 @@
-class Score {
-  constructor(id, name, score) {
-    this.id = id;
-    this.name = name;
-    this.score = score;
-  }
-}
-
-const addScore = (name, score, scoresList) => {
-  const scoreObj = new Score(scoresList.length, name, score);
-  scoresList.push(scoreObj);
-  localStorage.setItem('scores', JSON.stringify(scoresList));
+import { addGame } from './Game.js';
+
+// Create a new game using the leaderboard api
+const createGame = async () => {
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/', {
+    method: 'POST',
+    body: JSON.stringify({
+      name: 'New Chess Game',
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  const gameId = (await response.json()).result.substr(14, 20);
+  addGame(gameId);
+};
```

Comment:
- [ ] Please clean the list and apply the right methods as there are too many records and when I try to add new one and refresh, I can't see my record on the list.

Reply:
Hey @tugsem , thank you for the review. Though I want to ask how I can clean up the list , because this is the list of scores I am getting when I try to fetch using the game Id I created here. 
fetch(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores`);

Could you please help me ?